docs/HANDLED_RULES.md

```diff
@@ -141,6 +141,42 @@ Example:
 
 -----
 
+#### JEE applications should not "getClassLoader" ([Sonar Rule 3032](https://rules.sonarsource.com/java/RSPEC-3032))
+
+Using the standard getClassLoader() may not return the right class loader in Java Enterprise Edition context, i.e. the code use any of these [javax packages](https://docs.oracle.com/javaee/6/api/overview-summary.html). Instead, go through the currentThread. In particular, this processor will transform the code in `.java` files if they appear to use the `javax` package.
```

Comment:
Format getClassLoader() like `getClassLoader()` (and all source code elements for further PRs).